src/api.ts

```diff
@@ -16,19 +18,33 @@ type IncompleteRating = {
   }
 }
 
-type CompleteRating = {
-  attributes: {
-    avgScore: number;
-    rateableId: number;
-    rateableType: string;
-    userRatingsCount: number;
-  }
-}
+// Consolidated this into Rating above^
+
+// type CompleteRating = {
+//   attributes: {
+//     avgScore: number;
+//     rateableId: number;
+//     rateableType: string;
+//     userRatingsCount: number;
+//   }
+// }
 
 
 export const getTrending = async (): Promise<Trending> => {
-  const response = await fetch('https://speak-easy-staging.herokuapp.com/api/analytics/trending/atk')
-  const trending = await response.json()
-  return trending
+  try {
+  const response = await fetch('https://speak-easy-staging.herokuapp.com/api/analytics/trending/atk');
+
+  // add if statement to check for error
+  if(!response.ok) {
+    throw new Error('you did the error');
+  }
+
+  const trending = await response.json();
+
+  return trending;
+} catch (error) { console.log("you did the error again", error); throw error;
```

Comment:
While this is valid I think in this case the throw in the catch block is unnecessary.  See this: https://www.w3schools.com/js/js_errors.asp